Enh/add-changelog (#11)
callbellchat/callbell-public-docs: docs/release_notes.md

---
sidebar_position: 4
---

# Release notes

> A list of all the changes and enhancements that are introduced to our API. Use them to check whether new endpoints are added, or changes are made.

Reviewer: ```suggestion
A list of all the changes and enhancements that were introduced in our API. Use it to check whenever new endpoints are added, or changes are made.
```
